app/controllers/api/v1/api_controller.rb

```diff
@@ -10,10 +10,17 @@ class ApiController < ApplicationController
         :render_error_response
       rescue_from ActiveRecord::RecordInvalid, with:
         :render_error_response
+      rescue_from ActiveRecord::RecordNotFound, with:
+        :render_not_found_response
 
       def render_error_response(error)
         render json: { error: error }, status: 400
       end
+
+      def render_not_found_response(error)
+        render json: { error: I18n.t('api.error.invalid_request.content_not_found')
+          }, status: 404
```

Comment:
`status: :not_found`

[Rails HTTP Status Codes](http://www.railsstatuscodes.com/l)

Also you can edit `400` above